tools/block-generator/generator/config.go

```diff
@@ -70,15 +70,19 @@ const (
 	appBoxesClose  TxTypeID = "app_boxes_close"
 	appBoxesClear  TxTypeID = "app_boxes_clear"
 
-	// TODO: consider an app that creates/destroys an app during opup
+	// Special TxTypeID's recording effects of higher level transactions
+	effectPaymentTxSibling TxTypeID = "effect_payment_sibling"
+	effectInnerTx          TxTypeID = "effect_inner_tx"
```

Comment:
Are these configurable? I believe the `TxTypeID` was previously used  for the different types of transactions that would be chosen according to the ratios. This seems more like something that would be in `GenerationConfig` like `tx_per_block`.

Reply:
I think you're right that it's better to keep `TxTypeID` for types of transactions that can actually be chosen. I'll work to carve out a separate types for these _effects_.

Reply:
However, one issue here is that I do need to record the effects and these get stored in a `type Report map[TxTypeID]TxData` whose key is of type `TxTypeID`. So it does complicates things a bit. If we relax `Report` to have keys of type `string`, I still ought to be able to separate it out. I'll keep you posted.